modules/consts.go

```diff
@@ -0,0 +1,10 @@
+package modules
+
+import "github.com/NebulousLabs/Sia/types"
+
+// Consts that are required by multiple modules
+const (
+	// maxTxnAge determines the maximum age of a transaction (in block height)
+	// allowed before the transaction is pruned from the transaction pool.
+	MaxTxnAge = types.BlockHeight(24)
```

Comment:
I believe that there are other consts throughout the modules package, we may want to collect them all here

Reply:
Should that be done in the same PR?